droughtpipeline/load.py

```diff
@@ -0,0 +1,829 @@
+                                        triggered=(
+                                            True if event_type == "trigger" else False
+                                        ),
+                                        trigger_class=pipeline_will_trigger_portal,
+                                    )
+                                exposure_pcodes.append({"placeCode": pcode, "amount": amount})
+                                processed_pcodes.append(pcode)
+
+                            body = {
+                                "countryCodeISO3": country,
+                                "leadTime": f"{lead_time_event}-month",
+                                "dynamicIndicator": indicator,
+                                "adminLevel": int(adm_level),
+                                "exposurePlaceCodes": exposure_pcodes,
+                                "disasterType": disasterType,
+                                "eventName": event_name,
+                                "date": upload_time,
+                            }
+                            statsPath=drought_extent.replace(".tif", f"{indicator}_{lead_time_event}-month_{country}_{adm_level}.json" )
+                            with open(statsPath, 'w') as fp:
+                                    json.dump(body, fp)
+
+                            self.ibf_api_post_request("admin-area-dynamic-data/exposure", body=body )
+                    processed_pcodes = list(set(processed_pcodes))
+
+            # send trigger per lead time: event/triggers-per-leadtime
+            triggers_per_lead_time = [] 
+
+            for lead_time in range(0,5):#set(lead_times_list):
+                is_trigger, is_trigger_or_alert = False, False  
+                for lead_time_event, event_type in events.items():
+                    if event_type == "trigger" and lead_time  >= lead_time_event:
+                        is_trigger = True
+                    if (
+                        event_type == "trigger" or event_type == "alert"
+                    ) and lead_time >= lead_time_event:
+                        is_trigger_or_alert = True
+                triggers_per_lead_time.append(
+                    {
+                        "leadTime": f"{lead_time}-month",
+                        "triggered": is_trigger_or_alert,
+                        "thresholdReached": is_trigger,
+                    }
+                )
+            body = {
+                "countryCodeISO3": country,
+                "triggersPerLeadTime": triggers_per_lead_time,
+                "disasterType": disasterType,
+                "eventName": event_name,
+                "date": upload_time,
+            }
+            statsPath=drought_extent.replace(".tif", f"_{lead_time}-month_{country}.json" )
+            with open(statsPath, 'w') as fp:
+                    json.dump(body, fp)
+            self.ibf_api_post_request("event/triggers-per-leadtime", body=body)
```

Comment:
@aklilu just noticed this api-call, but this whole call can be removed. It is something floods-specific that is not needed for drought.

Reply:
done